app/depict/src/test/java/org/openscience/cdk/depict/DepictionTest.java

```diff
@@ -75,7 +75,7 @@ public void depictAsSvg() throws CDKException {
         SmilesParser sp = new SmilesParser(SilentChemObjectBuilder.getInstance());
         IAtomContainer ac = sp.parseSmiles("[nH]1cccc1");
         String svg = dg.depict(ac).toSvgStr();
-        String nl = System.getProperty("line.separator");
+        String nl = "\n";
```

Comment:
Mmm... this was probably there for a reason.

Reply:
Well, given that most (all?) of the development and testing happens on linux/mac machines and the test broke an my windows machine before switching **to** `\n` it seemed the better choice to give up on the assumed platform indepdence of the line separator 🤷🏼